### PR TITLE
Spotlight BTN added

### PR DESCRIPTION
**Pull request by mrSidSat**
### **_Spotlight btn as the name discussed in previous pr #967_** 

Issue: #883 

As discussed before all changes made and name changed to spotlight

### CODE REVIEW

**Comment by mrSidSat** [comment]
here Also **_HOVER BUTTON CHANGE BACKGROUND COLOR_** commit for the issue **#416** 
 commit (751d954)

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -159,6 +159,13 @@ $(document).ready(function () {
           label: "Aura Pulse",
           classes: "aurapulse-btn",
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:162]: Place this in the correct alphabetical order

**Review by shahednasser — CHANGES_REQUESTED** [review]
It all looks great I just have comments below regarding the code. I think the code in `_spotlight.less` has a lot of repeated code, which is like not using the benefits of Less. it can be fixed by properly nesting the rules inside the elements. Also please rename the button `spotlight-btn` for convention reasons (make sure to rename its file as well.)

As for the other files please create a different PR with it, don't combine issues in the same PR. But make sure that the style of the button is similar to "Outline button" but is filled on hover. Also make sure to call the button `colored-on-hover-btn` and place it under animated.

File: src/components/animated/_spotlight.less [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+	0% {
+		color: black;
+	}
+	100% {
+		color: cornsilk;
+		text-shadow: 0px 0px 20px white;
+		box-shadow: inset 0px 0px 5px cornsilk;
+	}
+}
+.sbtn.spotlight {
```
[the remark this answers is not in the document]
Comment (mrSidSat) [reply src/components/animated/_spotlight.less:45]: In the overall project all buttons have white color by default as stated in the .sbtn class .
but in the spotlight btn the background is transparent so the text should be black before the animation. which is not possible as the .sbtn is called first so is the dominant class causes it to be white . So i think it is okay to use .sbtn.spotlight-btn to make the property specific and according to need..

[the remark this answers is not in the document]
Comment (shahednasser) [reply src/components/animated/_spotlight.less:45]: Then add it for the `white-btn` section but not to the overall code

File: src/components/animated/_spotlight.less [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+	width: 200px;
+	background-color: transparent;
+	overflow: visible !important;
+	cursor: pointer;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	box-shadow: inset 0px 0px 10px #0fe0f5;
+}
+.spotlight {
```
[the remark this answers is not in the document]
Comment (mrSidSat) [reply src/components/animated/_spotlight.less:60]: the problem is I directly code in css so I have a hard time doing it in less..

[the remark this answers is not in the document]
Comment (shahednasser) [reply src/components/animated/_spotlight.less:60]: I understand @mrSidSat but unfortunately, the code needs to be in less.

[the remark this answers is not in the document]
Comment (mrSidSat) [reply src/components/animated/_spotlight.less:60]: ok

**Pull request closed without merging**